oak-security-spi/src/main/java/org/apache/jackrabbit/oak/spi/security/user/package-info.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@Version("2.5.0")
+@Version("2.6.0")
```

Comment:
imho this is not needed

Reply:
the additions to UserConstants required a version increase to `org.apache.jackrabbit.oak.spi.security.user` package